account_financial_report/report/aged_partner_balance_xlsx.py

```diff
@@ -0,0 +1,367 @@
+# Author: Julien Coux
+# Copyright 2016 Camptocamp SA
+# Copyright 2021 Tecnativa - Jo??o Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+
+
+class AgedPartnerBalanceXslx(models.AbstractModel):
+    _name = "report.a_f_r.report_aged_partner_balance_xlsx"
+    _description = "Aged Partner Balance XLSL Report"
+    _inherit = "report.account_financial_report.abstract_report_xlsx"
+
+    def _get_report_name(self, report, data=False):
+        company_id = data.get("company_id", False)
+        report_name = _("Aged Partner Balance")
```

Comment:
Looks like you forgot to replace it with `self.env._` for the files in the report folder